### PR TITLE
feat(dms): import dms resource, unit tests and docs

### PR DESCRIPTION
**Pull request by Zhukun-Huawei**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./g42cloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/g42cloud-terraform/terraform-provider-g42cloud/g42cloud       70.796s
```

### CODE REVIEW

File: docs/data-sources/dms_rabbitmq_flavors.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  + **dms.physical.storage.ultra.v2**: Type of the disk that uses ultra-high I/O.
+
+* `type` - (Optional, String) Specifies flavor type. The valid values are **single** and **cluster**.
+
+* `arch_type` - (Optional, String) Specifies the type of CPU architecture, e.g. **X86**.
+
+* `availability_zones` - (Optional, List) Specifies the list of availability zones with available resources.
+
+* `charging_mode` - (Optional, String) Specifies the flavor billing mode.
+  The valid values are **prePaid** and **postPaid**.
```
Comment (chengxiangdong) [inline docs/data-sources/dms_rabbitmq_flavors.md:59]: The billing modes are all `postPaid` and this field is not required.

File: docs/data-sources/dms_rabbitmq_flavors.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+* `id` - Indicates the flavor ID.
+
+* `type` - Indicates the flavor type.
+
+* `vm_specification` - Indicates the underlying VM specification.
+
+* `arch_types` - Indicates the list of supported CPU architectures.
+
+* `charging_modes` - Indicates the list of supported billing modes.
```
Comment (chengxiangdong) [inline docs/data-sources/dms_rabbitmq_flavors.md:83]: ditto

File: g42cloud/services/acceptance/dms/data_source_g42cloud_dms_rabbitmq_flavors_test.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+func testAccDatasourceDmsRabbitMQFlavors_basic() string {
+	return `
+data "g42cloud_dms_rabbitmq_flavors" "baisc" {
+  type = "cluster"
+}
+
+data "g42cloud_dms_rabbitmq_flavors" "test" {
+  type               = local.test_refer.type
+  arch_type          = local.test_refer.arch_types[0]
+  charging_mode      = local.test_refer.charging_modes[0]
```
Comment (chengxiangdong) [inline g42cloud/services/acceptance/dms/data_source_g42cloud_dms_rabbitmq_flavors_test.go:46]: ditto

File: docs/data-sources/dms_rabbitmq_flavors.md (hunk 1 above)
Comment (Zhukun-Huawei) [reply docs/data-sources/dms_rabbitmq_flavors.md]: done, fix it

File: docs/data-sources/dms_rabbitmq_flavors.md (hunk 2 above)
Comment (Zhukun-Huawei) [reply docs/data-sources/dms_rabbitmq_flavors.md]: done, fix it

File: g42cloud/services/acceptance/dms/data_source_g42cloud_dms_rabbitmq_flavors_test.go (hunk 3 above)
Comment (Zhukun-Huawei) [reply g42cloud/services/acceptance/dms/data_source_g42cloud_dms_rabbitmq_flavors_test.go]: done, fix it

**Pull request merged**